examples/table-queries/table_queries.bal

```diff
@@ -40,114 +40,77 @@ type PersonPublicProfile record {
-    Person[] personData = [p1, p2, p3, p4];
-
-    // This inserts the `Person` struct objects into the table and populates it.
-    foreach (p in personData) {
-        var ret = personTable.add(p);
-        match ret {
-            () => io:print("\nAdding to table successful");
-            error err => io:println("Adding to table failed: " + err.message);
-        }
-    }
+    // These are a few sample values that represent different `Person` records.
+    Person p1 = { id: 1, age: 25, salary: 1000.50, name: "jane", married: true };
+    Person p2 = { id: 2, age: 26, salary: 1050.50, name: "kane", married: false };
+    Person p3 = { id: 3, age: 27, salary: 1200.50, name: "jack", married: true };
+    Person p4 = { id: 4, age: 28, salary: 1100.50, name: "alex", married: false };
+
+    // This is the in-memory table that is constrained by the `Person` type.
+    table<Person> personTable = table {
+        data: [p1, p2, p3, p4]
+    };
 
     // This prints the `Person` table content.
-    io:print("\nThe personTable: ");
-    io:println(personTable);
+    printTable(queryStmt, "The personTable:  ", personTable);
 
-    // These are a few sample values that represent orders made by the people who were listed
-    //above as person structs.
+    // These are a few sample values that represent orders made by the people who were
+    // listed above as person records.
     Order o1 =
-        { personId: 1, orderId: 1234, items: "pen, book, eraser", amount: 34.75 };
+    { personId: 1, orderId: 1234, items: "pen, book, eraser", amount: 34.75 };
     Order o2 =
-        { personId: 1, orderId: 2314, items: "dhal, rice, carrot", amount: 14.75 };
+    { personId: 1, orderId: 2314, items: "dhal, rice, carrot", amount: 14.75 };
     Order o3 =
-        { personId: 2, orderId: 5643, items: "Macbook Pro", amount: 2334.75 };
-    Order o4 =
-        { personId: 3, orderId: 8765, items: "Tshirt", amount: 20.75 };
-    Order[] orderData = [o1, o2, o3, o4];
-
-    // This inserts the `Order` struct objects into the table and populates it.
-    foreach (o in orderData) {
-        var ret = orderTable.add(o);
-        match ret {
-            () => io:print("\nAdding to order successful");
-            error err => io:println("Adding to order failed: " + err.message);
-        }
-    }
+    { personId: 2, orderId: 5643, items: "Macbook Pro", amount: 2334.75 };
+    Order o4 = { personId: 3, orderId: 8765, items: "Tshirt", amount: 20.75 };
+
+    // This is the in-memory table that is constrained by the `Order` type.
+    table<Order> orderTable = table {
+        data: [o1, o2, o3, o4]
+    };
 
     // This prints the `Order` table content.
-    io:print("\nThe orderTable: ");
-    io:println(orderTable);
+    printTable(queryStmt, "The orderTable: ", orderTable);
 
     // Querying for a table always returns a new in-memory table.
 
     // 1. This queries for all the records in a table and returns them as another in-memory table.
```

Comment:
Can we remove the numbers? We do not seem to follow this format for other examples. WDYT?